tvtk/__init__.py

```diff
@@ -6,6 +6,10 @@
 """
 
 from os.path import exists, join, dirname, isdir
+import logging
+
+# use a null handler when used as a library
+logging.getLogger('tvtk').addHandler(logging.NullHandler())
```

Comment:
Thanks, do you think this link could be put in a comment above this?  Or is this obvious?